build/yuka.js

```diff
@@ -3125,7 +3125,11 @@
 		*
 		* @return {GameEntity} A reference to this game entity.
 		*/
-		start() {}
+		start() {
```

Comment:
Do you mind to undo the changes to the build files?

Reply:
done and force pushed.